CSharp/BatchExplorer/Models/TaskModel.cs

```diff
@@ -285,6 +327,39 @@ public override List<PropertyModel> PropertyModel
 
                     await asyncListSubtasksTask;
 
+                    // Linked Storage Account Files
```

Comment:
What is the reason for not using TaskOutputStorage.ListOutputs - is it because you want to be able to browse all uploaded files, not just by output kind?  If so we should consider adding an overload to the File Conventions library for this, rather than requiring you to write code directly against storage.